pycon-submission.md

```diff
@@ -12,87 +25,124 @@ Intermediate
 
 # Domain Level
 
-Intermediate
+Introductory
 
 # Description
 
-IPython and Jupyter provide tools for interactive and parallel computing that are widely
+IPython and Jupyter provide tools for interactive computing that are widely
 used in scientific computing, but can benefit any Python developer.
 
-We will show how to use IPython in different ways, as: an interactive shell,
-a graphical console, a network-aware VM in GUIs, a web-based
-notebook with code, graphics and rich HTML, and a high-level framework for
-parallel computing.
+We will show how to use IPython in different ways, as:
+
+- an interactive shell,
+- a graphical console,
+- a network-aware VM in GUIs,
+- a web-based notebook with code, graphics and rich HTML.
 
-# Audience 
+We will demonstrate how to deploy a custom environment
+with Docker that not only contains multiple Python kernels but also a couple
+of other languages.
+
+# Audience
 
 Programmers interested in using Python interactively, especially in data
-analysis environments.
+analysis environments. Prior knowledge of Python is best. Some experience
+with Docker for the last quarter of the tutorial would be a plus.
 
 # Objectives
 
-At the end of this tutorial, attendees will have an understanding of the overall design of Jupyter as a suite of applications they can use and combine in multiple ways in the course of their development work with Python and other languages. They will learn:
+At the end of this tutorial, attendees will have an understanding of the
+overall design of Jupyter (and IPython) as a suite of applications they can use
+and combine in multiple ways in the course of their development work with
+Python and other languages. They will learn:
 
 * Tricks from the IPython machinery that are useful in everyday development,
 
-* What the high-level applications in Jupyter, the web-based notebooks and the graphical Qt console, can do and how they can be used.
+* What the high-level applications in Jupyter, the web-based notebooks can do
+  and how they can be used.
 
-* How the IPython's architecture forms a natural foundation for high-level parallel computing with low latency and high throughput.
+* How the overall picture of IPython and Jupyter fits together, so that they
```

Comment:
* How to use IPython and Jupyter together so that they can be best used for the problem at hand.